### PR TITLE
Add Markup and styles for homepage of youtube clone

### PR DESCRIPTION
**Pull request by zmekonnen251**
# I added  the following files 
- index.html
- style.css
- linters for local and remote repositories

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  column-gap: 30px;
+}
+
+header {
+  padding: 40px;
+  grid-column: 1/3;
+  grid-row: 1/2;
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (zmekonnen251) [inline style.css:28]: Try to use CSS class for styling elements of HTML, don't target the element directly.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+nav li {
+  display: inline-block;
+  margin-right: 30px;
+}
+
+aside ul li {
+  display: flex;
+  gap: 30px;
+  margin-bottom: 30px;
+}
```
Comment (zmekonnen251) [inline style.css:43]: Try to use CSS class for styling elements of HTML, don't target the element directly.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+.search {
+  margin-top: 0;
+  height: 40px;
+}
+
+iframe {
+  width: 750px;
+  height: 400px;
+  margin-bottom: 20px;
+}
```
Comment (zmekonnen251) [inline style.css:92]: Try to use CSS class for styling elements of HTML, don't target the element directly.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+.channel-info {
+  margin-left: 30px;
+  padding: 0 30px;
+}
+
+h4 {
+  margin-left: 80px;
+}
```
Comment (zmekonnen251) [inline style.css:154]: Try to use CSS class for styling elements of HTML, don't target the element directly.

**Review by zmekonnen251 — COMMENTED** [review]
You are almost there @zmekonnen251, make changes for I commented

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @zmekonnen251, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# **Suggestions** you could apply in your next reviews 

- Please try not to sound as you are ordering `changes` remember it's just a suggestion even if you know it's 100% the correct thing
- Please try to provide the reason behind why YOU think change is crucial since it might sound obvious to you but not for others 
- Last but not least try to use some emojis to make your review look more friendly 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**